php/class-post-type.php

```diff
@@ -970,22 +1014,56 @@ public function merge_snapshots( $post_ids ) {
 		}
 		$conflict_keys = array_flip( $conflict_keys );
 		$merged_snapshot_data = call_user_func_array( 'array_merge', $snapshots_data );
+		$merged_keys = array_keys( array_diff_key( $merged_snapshot_data, $conflict_keys ) );
+
+		foreach ( $merged_keys as $key ) {
+			// Store contributed snapshot id in setting.
+			$uuid = array();
+			foreach ( $snapshot_post_data as $post_data ) {
+				if ( isset( $post_data['data'][ $key ]['merged_uuid'] ) && is_array( $post_data['data'][ $key ]['merged_uuid'] ) ) {
+					$uuid = array_merge( $uuid, $post_data['data'][ $key ]['merged_uuid'] );
+				} elseif ( isset( $post_data['data'][ $key ] ) ) {
+					$uuid[] = $post_data['uuid'];
+				}
+			}
+			$merged_snapshot_data[ $key ]['merged_uuid'] = array_values( array_unique( $uuid ) );
```

Comment:
Instead of `merged_uuid` what about `merged_changeset_uuids` or `merged_changesets`?

Reply:
Will use `merged_changeset_uuids` part of #93 I'll fix in it.